xml/deployment_expert_partitioner.xml

```diff
@@ -29,6 +29,45 @@
   option to use iSCSI as a networked disk<phrase os="sles"> (read more about
   iSCSI in <xref linkend="cha-iscsi"/>)</phrase>.
  </para>
+ <warning>
+  <title>Disk Space Units</title>
+  <para>
+   Note that for partitioning purposes, disk space is measured in binary
+   units, rather than in decimal units. For example, if you enter sizes of
+   <literal>1GB</literal>, <literal>1GiB</literal> or <literal>1G</literal>,
+   they all signify 1&nbsp;GiB (Gibibyte), as opposed to 1&nbsp;GB (Gigabyte).
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>Binary</term>
+    <listitem>
+     <para>
+      1&nbsp;GiB = 1&nbsp;073&nbsp;741&nbsp;824 bytes.
```

Comment:
Good stuff, this is one of those pitfalls that trap the unwary.

Why use spaces instead of commas?

Reply:
Good question. This is a standard "Brit abroad" precautionary measure. 

In non-English-speaking Europe, notably including Germany & Czechia, they use commas for the decimal marker, and sometimes, periods as the three-digit separator. So if I put in an expenses claim for €12.34 it will say €12,34 etc.  This confuses English speakers but our way confuses them, so it's common to replace the 3-digit separator with a non-breaking space.

I am not aware of house style but I should be -- I will find out.